### ISSUE TITLE
MapMaker's Fucked up

### ISSUE THREAD

**Issue by GeneralVeers25** [d3athrow/vgstation13#30580]
#### Description of issue

In the map maker, unless you hide turfs, lattices and catwalks aren't visible over space tiles. It's a small thing but hugely frustrating. I attached photos.

![turffuckup1](https://user-images.githubusercontent.com/40326633/132301634-848f4f7a-3ba6-4828-9c25-e443333dec52.PNG)

![turffuckup2](https://user-images.githubusercontent.com/40326633/132301640-fa67c195-335e-4010-8025-15c1232f026f.PNG)

#### Steps to reproduce

Use StrongDMM and open any map.

#### Length of time in which bug has been known to occur

Basically...since Multi-Z was merged I think. It wasn't like this when I started mapping Waystation, only started happening about halfway through.
[general] [qol]

**Comment by ShiftyRail** [comment]
✈️ 🪐

**Comment by ghost** [comment]
Most turfs and space structures (which should honestly just be turfs) are on plane -30000 while space is on -99, if that explains it.
